### ISSUE TITLE
vbInactiveTitleBarText enum missing

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#721]
**Describe the bug**
VBx did a naming error for `vbInactiveCaptionText` (&H800000**13**)
tB correctly replicated this wrongly named enum.

However, VBx corrected this by including a same valued enum `vbInactiveTitleBarText` (&H800000**13**)

**To Reproduce**
```
Dim i As VBRUN.SystemColorConstants
i = vbInactiveTitleBarText + vbInactiveCaptionText
```

**Expected behavior**
That tB includes `vbInactiveTitleBarText` with the same value as `vbInactiveCaptionText`

**Screenshots**
![image](https://user-images.githubusercontent.com/33756666/153197423-0afac81d-2aa2-40bd-944d-7c15d42e7624.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.14.10]

**Additional context**
n/a

**Comment by WaynePhillipsEA** [comment]
It looks like there's a few other missing too.   The tB ones were copied from the VBA version of SystemColorConstants, which doesn't include them at all.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.14.11, thanks!
